docs/source/en/quantization.md

```diff
@@ -745,3 +745,43 @@ The speed and throughput of fused and unfused modules were also tested with the
     <figcaption class="mt-2 text-center text-sm text-gray-500">generate throughput/batch size</figcaption>
   </div>
 </div>
+
+## HQQ 
+Half-Quadratic Quantization (HQQ) implements on-the-fly quantization via fast robust optimization. It doesn't require calibration data and can be used to quantize any model.  
+Please refer to the <a href="https://github.com/mobiusml/hqq/">official package</a> for more details.
+
+For installation, we recommend you use the following approach to get the latest version and build its corresponding CUDA kernels:
+```
+pip install hqq
+```
+
+To quantize a model, you need to create an ```HqqConfig``` as follows:
+``` Python
+from transformers import AutoModelForCausalLM, AutoTokenizer, HqqConfig
+
+#Linear layers will use the same quantization config
+quant_config  = HqqConfig(nbits=8, group_size=64, quant_zero=False, quant_scale=False, axis=0) #axis=0 is used by default
+
+#Each type of linear layer (referred to as linear tag) will use different quantization parameters
```

Comment:
Just to make sure I've understood - is this showing two different ways to create the config? If so, we hsould try to clarify this a bit e.g.: 
* Comments refer to what someone can do e.g. `Create the config such that each linear layer will use ...`
* Explicitly mention in the docs that there's two ways of instantiating the config
* Explain why you might want to do one over the other

If that's not the case, then we should explain as bit more what this example is showing

Reply:
https://github.com/huggingface/transformers/pull/29637/commits/3522f0a672a31f9f3697f2814ec4574ae39f9a34